test/pkg/environment/common/expectations.go

```diff
@@ -118,16 +148,14 @@ func (env *Environment) ExpectSettingsReplaced(data ...map[string]string) {
 	}
 	// Update the configMap to update the settings
 	env.ExpectCreatedOrUpdated(cm)
-
-	// Get the karpenter pods and delete them to restart the containers
-	env.ExpectKarpenterPodsDeletedWithOffset(1)
-	env.EventuallyExpectKarpenterPodsHealthyWithOffset(1)
```

Comment:
Maybe I'm missing where this is plugged in, but don't we still need to restart the deployment to get the configmap changes?

Reply:
Yep, just re-added it back since I realized that I missed this